enterprise/frontend/src/embedding-sdk/hooks/private/use-sdk-usage-problem.ts

```diff
@@ -17,15 +17,15 @@ export function useSdkUsageProblem({
 }) {
   const hasLoggedRef = useRef(false);
 
-  const dispatch = useDispatch();
+  const dispatch = useSdkDispatch();
 
   // When the setting haven't been loaded or failed to query, we assume that the
   // feature is _enabled_ first. Otherwise, when a user's instance is temporarily down,
   // their customer would see an alarming error message on production.
   // TODO: replace this with "enable-embedding-sdk" once the settings PR landed.
   const isEnabled = useSetting("enable-embedding") ?? true;
 
-  const hasTokenFeature = useSelector(state => {
+  const hasTokenFeature = useSdkSelector(state => {
```

Comment:
Just curious, should we always use `useSdkSelector` even when we are not referencing sdk store states? Or is this mostly for consistency.

Reply:
We have a rule for useSdkDispatch: https://github.com/metabase/metabase/blob/09a4f60666594eca8bfd56e235502ca887909445/enterprise/frontend/src/embedding-sdk/.eslintrc#L6-L11
So I updated the import of the selector too for consistency, we should probably add it to the lin rule too